app/controllers/tt/api/v1/visitor_details_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Tt::Api::V1::VisitorDetailsController < Tt::Api::V1::ApplicationController
+  def show
+    render json: TestTrack::FakeServer.visitor_details
```

Comment:
I was thinking this morning that this still has a bunch of unnecessary indirection, when really we could just have a `show.json`. but I guess it makes sense to keep it consistent with the others and centralize around `FakeServer`.